api/src/pcapi/core/token/__init__.py

```diff
@@ -114,19 +116,29 @@ class Token(AbstractToken):
     def load_without_checking(cls, encoded_token: str, *args: typing.Any, **kwargs: typing.Any) -> "Token":
         try:
             payload = utils.decode_jwt_token(encoded_token)
+        except jwt.exceptions.ExpiredSignatureError:
+            raise users_exceptions.ExpiredToken()
         except jwt.PyJWTError:
-            raise InvalidToken()
+            raise users_exceptions.InvalidToken()
         try:
             data = payload["data"] if payload["data"] is not None else {}
             type_ = TokenType(payload["token_type"])
             user_id = payload["user_id"]
             return cls(type_, user_id, encoded_token, data)
         except (KeyError, ValueError):
-            raise InvalidToken()
+            raise users_exceptions.InvalidToken()
 
     @classmethod
     def create(cls, type_: TokenType, ttl: timedelta | None, user_id: int, data: dict | None = None) -> "Token":
-        encoded_token = utils.encode_jwt_payload({"token_type": type_.value, "user_id": user_id, "data": data})
+        payload: dict[Any, Any] = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏a priori on sait ce que sont les clés
```suggestion
        payload: dict[str, Any] = {
```